### PR TITLE
Feat/add article task runner robo

### PR DESCRIPTION
**Pull request by breizh81**
> Article that speaks about the task-runner called robo

### CODE REVIEW

**Comment by ElevenWilson** [comment]
16 words might be misspelled, please check them: développpeurs, d'executer, ecrit, executer, GitHub, Kontomarkos, login, Packagist, phar, plusieures, possibillité, robo, Robo, RoboFile, skontomarkos, Symfony

**Comment by ElevenWilson** [comment]
12 words might be misspelled, please check them: développpeurs, GitHub, Kontomarkos, login, Packagist, phar, possibillité, robo, Robo, RoboFile, skontomarkos, Symfony

**Comment by ElevenWilson** [comment]
18 words might be misspelled, please check them: Blr, customisable, decidé, Deployer, développpeurs, GitHub, Kontomarkos, login, Packagist, phar, PHPUNIT, possibillité, robo, Robo, RoboFile, skontomarkos, supeviser, Symfony

**Comment by ElevenWilson** [comment]
18 words might be misspelled, please check them: Blr, customisable, decidé, Deployer, développpeurs, GitHub, Kontomarkos, login, Packagist, phar, PHPUNIT, possibillité, robo, Robo, RoboFile, skontomarkos, supeviser, Symfony

File: _authors/skontomarkos.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: author
+login: skontomarkos
+title: Sophie Kontomarkos
+twitter:-
```
Comment (snroki) [inline _authors/skontomarkos.md]:
```suggestion
```

**Comment by ElevenWilson** [comment]
18 words might be misspelled, please check them: Blr, customisable, decidé, Deployer, développpeurs, GitHub, Kontomarkos, login, Packagist, phar, PHPUNIT, possibillité, robo, Robo, RoboFile, skontomarkos, supeviser, Symfony

**Comment by ElevenWilson** [comment]
15 words might be misspelled, please check them: Blr, customisable, decidé, Deployer, développpeurs, GitHub, Packagist, phar, PHPUNIT, possibillité, robo, Robo, RoboFile, skontomarkos, supeviser

**Pull request closed without merging**